distributed/scheduler.py

```diff
@@ -796,6 +796,9 @@ class TaskGroup:
     _nbytes_in_memory: Py_ssize_t
     _duration: double
     _types: set
+    _start: double
```

Comment:
@jakirkham just wanted to note that we're not adding corresponding `property`s here like 

```python
@property
def start(self):
    return self._start
```

I think this should be okay as IIUC these `property`s were added so we could gradually cythonize the scheduler's internals. As `TaskGroup._start`, `._stop`, and `._all_durations` are new attributes, we don't need to worry about being backwards compatible in the same way. Please feel free to correct me if that's not the case

Reply:
Unfortunately that could result in breakage when Cythonizing. `@property` is needed for pure Python (non-typed) access of attributes

Reply:
I've added properties